test/channel_test.rb

```diff
@@ -0,0 +1,24 @@
+require_relative "test_helper"
+require_relative "../lib/channel"
+
+describe "Channel class" do
```

Comment:
Since a channel is a recipient, you should probably also test sending a message and the print details as well.